pkg/scheduler/cache/cache.go

```diff
@@ -640,6 +640,13 @@ func newSchedulerCache(config *rest.Config, schedulerNames []string, defaultQueu
 		UpdateFunc: sc.UpdateNumaInfoV1alpha1,
 		DeleteFunc: sc.DeleteNumaInfoV1alpha1,
 	})
+	// explictly register informers to the factory, otherwise resources listers cannot get anything
+	// even with no erorr returned.
+	informerFactory.Core().V1().Namespaces().Informer()
```

Comment:
Only selectorSpread plugin and topology spread plugin need below informer register?

Reply:
The `InterPodAffinity` plugin need the `namespace` informer, `SelectorSpread` and `PodTopologySpread`  needs the following four.

Reply:
> @elinx please merge the two commits to one.

done